repo_tests/ansible_inventory_path_exists_test.py

```
@@ -0,0 +1,171 @@
+def _unquote(token: str) -> str:
+    if len(token) >= 2 and token[0] == token[-1] and token[0] in ("'", '"'):
+        return token[1:-1]
+    return token
+
+
+def _ansible_inventory_refs(text: str) -> list[str]:
+    """Literal `-i <path>` arguments on live (non-comment, non-echoed) lines."""
+    refs: list[str] = []
+    for line in text.splitlines():
+        stripped = line.strip()
+        if not stripped or stripped.startswith("#") or stripped.startswith("echo"):
```

Ruff (0.16.3)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)